src/styles.css

```diff
@@ -156,14 +156,15 @@
   background: var(--blue-denim-500);
   border: 2px solid var(--blue-denim-600);
 }
-.cio-question-cta-button:active {
+.cio-question-cta-button:focus {
```

Comment:
I'm not entirely sure if it's this change but it makes the button stay "pressed" after I click and move out
![image](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/aa8b39e1-85a7-458c-9f59-50b595dad3f9)